python/lsst/pipe/tasks/postprocess.py

```diff
@@ -798,10 +799,31 @@ class TransformObjectCatalogConfig(TransformCatalogBaseConfig,
         doc=("Whether results dataframe should have a multilevel column index (True) or be flat "
              "and name-munged (False).")
     )
+    goodFlags = pexConfig.ListField(
+        dtype=str,
+        default=[],
+        doc=("List of 'good' flags that should be set False when filling empty tables. "
+             "All other flags are considered to be 'bad' flags and will be set to True.")
+    )
+    floatFillValue = pexConfig.Field(
+        dtype=float,
+        default=np.nan,
+        doc="Fill value for float fields when filling empty tables."
```

Comment:
Totally subjective: "Fill value...when filling" grates a tad.  Perhaps "Fill value...when populating"?  (totally up to you, though!  But if you do change, update the other two docs as well.)